zomes/dnahub_csr/Cargo.toml

```diff
@@ -12,6 +12,6 @@ crate-type = ["cdylib", "rlib"]
 dnahub = { path = "../dnahub" }
 hc_devhub_sdk = { version = "0.1", path = "../../devhub_sdk" }
 hc_dnahub_sdk = { version = "0.2", path = "../../dnas/dnahub/sdk" }
-hc_portal_sdk = "0.9"
+portal_sdk = { git = "https://github.com/holochain/portal-dna.git", package = "hc_portal_sdk", branch = "holochain-0.4.1" }
```

Comment:
Using git dependency due to missing permissions to publish to crates.io. Will be changed once permissions are granted